grakn-kb/src/test/java/ai/grakn/kb/internal/concept/EntityTypeTest.java

```diff
@@ -126,6 +127,20 @@ public void whenGettingTheSuperSet_ReturnAllOfItsSuperTypes() throws Exception{
     }
 
     @Test
+    public void whenGettingTheSuperSetViaSupsMethod_ReturnAllOfItsSuperTypes(){
```

Comment:
This is the test you should keep I think.

Reply:
Indeed.